src/pages/patientView/patientHeader/GenomeDrivenDiagnosis.tsx

```diff
@@ -0,0 +1,85 @@
+import * as React from "react";
+import DefaultTooltip from "shared/components/defaultTooltip/DefaultTooltip";
+import {placeArrowBottomLeft} from "shared/components/defaultTooltip/DefaultTooltip";
+import SimpleTable from "shared/components/simpleTable/SimpleTable";
+import FontAwesome from "react-fontawesome";
+
+export type GDDOutput = {
+    Pred1: string;
+    Pred2: string;
+    Pred3: string;
+    Conf1: string;
+    Conf2: string;
+    Conf3: string;
+}
+
+interface IGenomeDrivenDiagnosisProps {
+    prediction: GDDOutput;
+    sampleCancerType?: string;
+}
+
+export default class GenomeDrivenDiagnosis extends React.Component<IGenomeDrivenDiagnosisProps, {}> {
+    public render() {
+        return (
+            <DefaultTooltip
+                placement='bottomLeft'
+                trigger={['hover', 'focus']}
+                overlay={this.tooltipContent()}
+                arrowContent={<div className="rc-tooltip-arrow-inner" />}
```

Comment:
I think we don't need the `arrowContent` anymore, it should already be defined in the `DefaultTooltip`.